docs/static/breaking-changes.asciidoc

```diff
@@ -1,26 +1,200 @@
+7.x) is not recommended.
 
 See these topics for a description of breaking changes:
 
+* <<breaking-7.0>>
 * <<breaking-pq>>
 * <<breaking-6.0>>
 
 See also <<releasenotes>>. 
 
+[[breaking-7.0]]
+=== Breaking changes in 7.0
+
+coming[7.0.0]
+
+Here are the breaking changes for {ls} 7.0. 
+
+[float]
+==== Changes in Logstash Core
+
+These changes can affect any instance of Logstash that uses impacted features.
+Changes to Logstash Core are plugin agnostic.
+
+[float]
+===== Field Reference parser is more strict
+
+The Field Reference parser, which is used to interpret references to fields in
+your pipelines and plugins, was made to be more strict and will now reject
+inputs that are either ambiguous or illegal. Since 6.4, Logstash has emitted
+warnings when encountering input that is ambiguous, and allowed an early opt-in
+of strict-mode parsing either by providing the command-line flag
+`--field-reference-parser STRICT` or by adding `config.field_reference.parser:
+STRICT` to `logstash.yml`.
+
+[float]
+===== Java execution engine enabled by default
+
+The new Java execution engine is enabled by default. It features faster
+performance, reduced memory usage, and lower config startup and reload times.
+
+For more information, see the blog post about the
+https://www.elastic.co/blog/meet-the-new-logstash-java-execution-engine[initial
+release of the Java execution engine].  
+
+We went to considerable lengths to make this change seamless. Still, it's a big
+change. If you notice different behaviors that might be related, please let us
+know.
+
+[float]
+===== Beats conform to the Elastic Common Schema (ECS)
+
+As of 7.0, Beats fields conform to the {ecs-ref}/index.html[Elastic Common
+Schema (ECS)].
+
+If you upgrade Logstash before you upgrade Beats, the payloads continue to use
+the pre-ECS schema. If you upgrade your Beats before you upgrade Logstash, then
+you'll get payloads with the ECS schema in advance of any Logstash upgrade.
+
+If you see mapping conflicts after upgrade, that is an indication that the
+Beats/ECS change is influencing the data reaching existing indices. 
+
+See the *{beats} Platform Reference* for more information on
+{beats-ref}/upgrading-6-to-7.html#enable-ecs-compatibility[Beats and ECS].
+  
+[float]
+==== Changes in Logstash Plugins
+
+With 7.0.0, we have taken the opportunity to upgrade a number of bundled plugins
+to their newest major version, absorbing their breaking changes into the
+Logstash distribution.
+
+While these upgrades included new features and important fixes, only the
+breaking changes are called out below.
+
+NOTE: The majority of the changes to plugins are the removal of previously-deprecated
+and now-obsolete options. Please ensure that your pipeline
+configurations do not use these removed options before upgrading.
+
+Please review the 
+https://www.elastic.co/support/matrix#matrix_logstash_plugins[Support Matrix for Logstash plugins]
+for the latest on supported plugins.
+
```

Comment:
I can take out this link if y'all don't think it is needed or appropriate.

Reply:
yes let's remove this reference to supported plugins please